atomic_reactor/util.py

```diff
@@ -405,7 +351,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                 shutil.rmtree(self.our_tmpdir)
 
     def reset(self, git_reference):
-        self._commit_id = reset_git_repo(self.git_path, git_reference)
+        self._commit_id, _ = reset_git_repo(self.git_path, git_reference)
```

Comment:
pass self._git_depth to reset_git_repo ?

Reply:
That functionality is added in #1158

Reply:
ohhh ic